src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+.reset {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+  min-width: 800px;
+}
+
+.link {
+  display: flex;
+  padding-left: 50px;
+  text-decoration: none;
+}
+
+.link__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  margin: 0 50px 0 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  gap: 20px;
+}
+
+.nav__link {
+  --height: 60px;
+
+  height: var(--height);
+  line-height: var(--height);
+  text-align: center;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  color: rgba(0, 0, 0, 1);
+  text-transform: uppercase;
+  display: inline-block;
+  text-decoration: none;
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
```

Comment:
please check the color in the mock design it is #00acdc

Reply:
I guess it is same color. Before i used RGB, changed it to HEX according to your comments.
![image](https://user-images.githubusercontent.com/106464335/202147815-0d6b42f5-b577-4717-9512-de4fe3048e84.png)